docs/ci-operations-group.md

```diff
@@ -0,0 +1,140 @@
+# KubeVirt CI Operations Group
+
+
+## Goal of this document
```

Comment:
Would it be useful to also include information on who to contact or which meeting to attend if someone wants to be involved?

Reply:
Are you suggesting to move [this paragraph](https://github.com/kubevirt/project-infra/pull/2315/files#diff-1ecaecfe9bbe8e3852fccc6b8cb7da71400a7218f74ca7ba163e22fe0d35e396R126) to the top of the document?

Reply:
Not so much - just wondering if it makes sense to have a specific pointer for someone who was new and interested in helping. What would be the best first/next step for them? Are there issues to take, should the or should they attend a meeting or contact someone?
A line like "If you would like to contribute or be involved, then <insert next step here>" 
Either at the start below the goal, or as an intro to the meetings as you indicated.

Reply:
I've added it to the top of the document, WDYT?